java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -345,6 +346,12 @@ public CompletableFuture<Long> strlen(@NonNull String key) {
         return commandManager.submitNewCommand(Strlen, new String[] {key}, this::handleLongResponse);
     }
 
+    @Override
+    public CompletableFuture<Long> setrange(@NonNull String key, int offset, @NonNull String value) {
```

Comment:
There's no need to return long. Redis returns int, and it specifies in the documentation that the max offset is 536870911, which is also true for the return value (+1)